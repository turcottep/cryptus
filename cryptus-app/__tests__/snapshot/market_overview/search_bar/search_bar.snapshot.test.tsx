import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import SearchBar from "../../../../components/market_overview/search_bar/search_bar";

describe("<SearchBar />", () => {
  it("should display properly", async () => {
    const tree = renderer.create(<SearchBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
