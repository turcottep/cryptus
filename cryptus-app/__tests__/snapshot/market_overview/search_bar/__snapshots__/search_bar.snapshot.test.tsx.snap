// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBar /> should display properly 1`] = `
<div>
  <div>
    <img
      src="magnifier.svg"
    />
    <input
      placeholder="Search"
      type="text"
    />
  </div>
</div>
`;
